app/src/main/java/com/babylon/wallet/android/domain/model/SecurityProblem.kt

```diff
@@ -0,0 +1,31 @@
+package com.babylon.wallet.android.domain.model
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.babylon.wallet.android.R
+
+sealed interface SecurityProblem {
+    data class EntitiesNeedBackup(
+        val factorSourceID: String,
+        val accountsNeedBackup: Int,
+        val personasNeedBackup: Int
+    ) : SecurityProblem
+
+    data class EntitiesNeedRecovery(val factorSourceID: String) : SecurityProblem
+
+    data object BackupNotWorking : SecurityProblem
+}
+
+@Composable
+fun SecurityProblem.toProblemHeading(): String {
+    return when (this) {
+        is SecurityProblem.EntitiesNeedBackup -> stringResource(
+            id = R.string.securityCenter_problem3_heading,
+            accountsNeedBackup,
+            personasNeedBackup
+        )
+
+        is SecurityProblem.EntitiesNeedRecovery -> stringResource(id = R.string.securityCenter_problem9_heading)
+        SecurityProblem.BackupNotWorking -> stringResource(id = R.string.securityCenter_problem6_heading)
+    }
+}
```

Comment:
can we move this out of the domain package?
I think the `SettingsItem` file looks like the right place? Or as a separate file under the presentation/settings/ ?

Reply:
I don't think SettingsItem is the right place. Will move it to SecurityCenterScreen